src/NDItk/multigroup/CrossSection/src/write.hpp

```diff
@@ -0,0 +1,42 @@
+/**
+ *  @brief Write the record data
+ *
+ *  This assumes that the record is not empty.
+ *
+ *  @param[in] iter   the current position in the output
+ */
+template< typename OutputIterator >
+void write( OutputIterator& iter ) const {
+
+  std::ostringstream buffer;
+
+  auto x = this->begin();
+  buffer << "    " << x[0] << ' ' << std::setprecision( 15 ) << x[1] << '\n';
```

Comment:
It should be an integer in this case (it is the MT number), although a static_cast<int>( std::round(  ) ) may be in order here just in case. I'll modify fix/review.

Reply:
Good idea!

Reply:
Done. Added to fix/review